src/main/resources/application.properties

```diff
@@ -5,5 +5,4 @@ spring.datasource.password=kodilla_password
 spring.jpa.database=mysql
 spring.jpa.show-sql=true
 spring.jpa.hibernate.ddl-auto=update
-spring.jpa.hibernate.naming.strategy=org.hibernate.cfg.ImprovedNamingStrategy

```

Comment:
a czemu to usuwasz ?

Reply:
W poprzednich projektach nie działało mi, jak strategy było dodane, ale nie pamiętam, jakie błędy wyrzucało

Reply:
jak póki co nie wywala się to bym zostawił :)

Reply:
W intelliji pokazuje że to jest przestarzałe więc nie wiem czy jest sens zostawiać...

Reply:
Na razie z tym projekt się buduje, ale potem problem był na etapie uruchomienia aplikacji :-)